src/Tests/Microsoft.Diagnostics.Monitoring.Tool.FunctionalTests/ParameterCapturingTests.cs

```diff
@@ -131,23 +130,52 @@ public async Task StopsProducingLogStatementsAfterOperationCompletes(Architectur
                 await appRunner.SendCommandAsync(TestAppScenarios.ParameterCapturing.Commands.Validate);
             });
         }
-#else // NET7_0_OR_GREATER
+
         [Theory]
         [MemberData(nameof(ProfilerHelper.GetArchitecture), MemberType = typeof(ProfilerHelper))]
-        public async Task Net6AppFailsOperation(Architecture targetArchitecture)
+        public async Task CapturesParameters(Architecture targetArchitecture)
         {
-            await RunTestCaseCore(TestAppScenarios.ParameterCapturing.SubScenarios.AspNetApp, targetArchitecture, async (appRunner, apiClient) =>
+            await RunTestCaseCore(TestAppScenarios.ParameterCapturing.SubScenarios.DoNotExpectLogStatement, targetArchitecture, async (appRunner, apiClient) =>
```

Comment:
See my above comment. Given the size of this PR I'm fine if you want to revert changes to these files, skip the tests, and rework these tests in a seperate PR.

Reply:
This test verifies the capture output.
I kept or modified the loging tests because we technically still have access to an ILogger object and we might accidently log. I was planning on removing them once we remove the HostingStartup logic completly.

Reply:
> I kept or modified the loging tests because we technically still have access to an ILogger object and we might accidently log.

Sure but we technically have access to everything inside the process so I'm still not sure what we're gaining here -- we don't do anything with the ILogger in the probes so this test isn't doing anything of value.  Given the fact that we're planning on removing the ILogger altogether and this PR is already a bit too unweidly in size to properly review let's just disable these tests and rework them in a separate PR

Reply:
I reverted the tests and will refactor them in a separate PR